lib/rialto/etl/cli.rb

```diff
@@ -2,15 +2,17 @@
 
 require 'thor'
 require 'rialto/etl'
-require 'rialto/etl/cli/extract'
-require 'rialto/etl/cli/transform'
-require 'rialto/etl/cli/load'
-require 'rialto/etl/cli/grants'
-require 'rialto/etl/cli/publications'
 
 module Rialto
   module Etl
     # A module to hold command-line interface classes
     module CLI; end
   end
 end
+require 'active_support/core_ext/module/delegation'
```

Comment:
Can you drop a comment here about why only some of the `require` statements have moved beneath the `module` declaration?

Reply:
Because they are all children of `Riato::Etl::CLI` so it's silly to add the children before adding the `CLI` module.